spec/controllers/posts_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe PostsController do
+  describe 'GET #index' do
+    let!(:post) { FactoryGirl.create(:post) }
+    before { get :index }
+
+    it { is_expected.to render_template 'index' }
+
+    it 'assigns @posts' do
+      expect(assigns :posts).to eq [post]
+    end
+  end
+
+  describe 'GET #new' do
+    before { get :new }
+
+    it { is_expected.to render_template 'new'}
+
+    it 'assigns a new Post to @post' do
+      expect(assigns :post).to be_a_new(Post)
+    end
+  end
+
+  describe 'POST #create' do
+    context 'with valid attributes' do
+      it 'creates a new post' do
+        expect{
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)